submissions/jeffersonMormah/html/contact.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
@Jeffbuchi well done. Your contact form is suppose to be in htmlForm folder.

Reply:
I guess I should remove the code on line one and recreate a folder called htmlForm?

Reply:
I guess I should remove the code on line 1 and recreate a folder called htmlForm instead of HTML @ToluAkin?

Reply:
@Jeffbuchi `contact.html` should be in htmlForm folder.

Reply:
@ToluAkin, Please Review changes. Thanks

Reply:
Well done. Remove the contact page from the html folder  and leave contact page in htmlForm folder @Jeffbuchi